Lib/test/test_codecs.py

```diff
@@ -858,7 +864,8 @@ def test_bug691291(self):
         self.addCleanup(os_helper.unlink, os_helper.TESTFN)
         with open(os_helper.TESTFN, 'wb') as fp:
             fp.write(s)
-        with warnings_helper.check_warnings(('', DeprecationWarning)):
+        # TODO: RUSTPYTHON - warnings_helper function not present
+        # with warnings_helper.check_warnings(('', DeprecationWarning)):
```

Comment:
We prefer not to change inner code when possible.
Since we already marked this test as failing test, this will be fine.
```suggestion
        with warnings_helper.check_warnings(('', DeprecationWarning)):
```